### ISSUE TITLE
v0.14.1 release

### ISSUE THREAD

**Issue by hdgarrood** [purescript/purescript#4054]
v0.14.0 was released on March 1 (at least, going by the Discourse announcement). March 1 was exactly 6 weeks ago, which means we are due a new release (aside: we should probably write something up about our release schedule somewhere).

The only things I'd still like to merge before we release are #4053 and #4047. Are there any other candidates we should consider? In general I think it's better to keep the 6 week cycle on track than to delay releases to get particular changes in, so I think we should err on the side of saying that things will have to wait until v0.14.2 if in doubt.

**Comment by hdgarrood** [comment]
I've just tagged v0.14.1, and I'll make the announcements once everything is built and released. :tada:

**Comment by mitchblank** [comment]
Is there something odd going on with hackage?  I see that you made a metadata revision for the 0.14.0 version a couple hours ago ( http://hackage.haskell.org/package/purescript-0.14.0/revisions/ ) but not an actual 0.14.1 version

Is the 0.14.1 update still pending on hackage or did something go wrong.  I only ask because I see that metadata change today which makes me wonder if some wires got crossed.

I have an update to homebrew ready to go which seems to work on the 0.14.1 tarball currently available on github ( https://github.com/Homebrew/homebrew-core/compare/master...mitchblank:purescript-0.14.1?expand=1 ) but since that formula normally uses hackage as its upstream I'm just waiting for the tarball to appear there before pulling the trigger.

**Comment by hdgarrood** [comment]
Nothing odd going on with Hackage, I just haven't uploaded the packages there yet. I made the metadata revision because I am about to release purescript-ast and purescript-cst on Hackage and I don't want 0.14.0 pulling the newer versions in because they have not been tested together.

**Comment by mitchblank** [comment]
OK, standing by.

**Comment by hdgarrood** [comment]
I've now uploaded all of the packages to Hackage and everything seems to have worked ok :)

**Comment by mitchblank** [comment]
Great.  I opened a homebrew PR here https://github.com/Homebrew/homebrew-core/pull/75429

(Someone else beat me to it with another PR, but theirs didn't include the hpack changes so I don't think their PR worked)
